hue2mqtt/mqtt/wrapper.py

```diff
@@ -80,6 +80,10 @@ async def connect(self) -> None:
         if self._broker_info.force_protocol_version_3_1:
             mqtt_version = gmqtt.constants.MQTTv311
 
+        if self._broker_info.enable_auth:
+            LOGGER.debug("MQTT Auth enabled")
+            self._client.set_auth_credentials(self._broker_info.username,self._broker_info.password)
```

Comment:
Linter has failed here :slightly_frowning_face: 

```
hue2mqtt/mqtt/wrapper.py:85:73: E231 missing whitespace after ','
hue2mqtt/mqtt/wrapper.py:85:91: E501 line too long (100 > 90 characters)
```

```suggestion
        if self._broker_info.enable_auth:
            LOGGER.debug("MQTT Auth enabled")
            self._client.set_auth_credentials(
                self._broker_info.username,
                self._broker_info.password,
            )
```